types/geo-coordinates-parser/package.json

```
@@ -0,0 +1,17 @@
+{
+    "private": true,
+    "name": "@types/geo-coordinates-parser",
+    "version": "1.6.9999",
```

```suggestion
    "version": "1.6.9999",
    "type": "module",
```

must match the implementation package: https://unpkg.com/browse/geo-coordinates-parser@1.6.3/package.json